src/Main.css

```diff
@@ -43,3 +43,8 @@ button, .ant-collapse-header, .react-geo-titlebar {
       min-width: unset !important;
   }
 }
+
+.ant-tooltip .ant-tooltip-inner,
+.ant-tooltip .ant-tooltip-arrow-content {
+  background-color: rgba(0,0,0,.5);
```

Comment:
So this will only work with white text color i guess. I think we need another variable for text background - in the future. I think its ok for now

Reply:
Since tooltip text is now excluded from css overrides (tooltip text always stays default like coming from antd), it should be ok here - tooltips are not affected from customized style.